aggregation/simple/src/test/java/com/spotify/heroic/aggregation/simple/AggregationSerializationTest.java

```diff
@@ -1,10 +1,14 @@
 package com.spotify.heroic.aggregation.simple;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotSame;
```

Comment:
are the new imports used in here?

Reply:
Good catch, I removed the unused import statements